Added info about geo support (#4452)
SkylineCommunications/dataminer-docs: dataminer-overview/DataMiner_Hosting_and_HA/DaaS_hosting.md

1. you create an organization,
1. you give a name to your DataMiner System (DMS), and
1. you can start adding elements to it, all with a couple of clicks.

# Regional Support for DaaS  

By default, your DaaS system is hosted in the Azure West Europe region. However, you can request deployment in [other Azure regions](https://datacenters.microsoft.com/globe/explore/).  

Geographically distributed systems with multiple nodes across different regions are also supported. If you require a *custom setup*, contact us at [daas@dataminer.services](mailto:daas@dataminer.services).  

> The linked storage (STaaS) for your DaaS cluster is hosted at a fixed location. For more details on STaaS Azure Storage location offerings, refer to our documentation: [Data Location and Redundancy](https://docs.dataminer.services/user-guide/Advanced_Functionality/Databases/STaaS/STaaS_features.html#data-location-and-redundancy)

Reviewer: This might sound like all the DaaS systems around the world are connected to one fixed location. Maybe we have to rephrase this a bit? Something like ...is hosted at the chosen location, and all the DMAs in the same cluster connect to the same location. Or something like that...

Reviewer 2: I think it might be better if we put less info about STaaS here and have the user click through to the "Data location and redundancy" page for all the info. Alternatively, we'd have to explain in a bit more detail to avoid causing confusion (e.g. what is the "chosen location"?), but I'm afraid we may end up with duplicate info then.
```suggestion
For information about the regional support for the linked storage (STaaS) for your DaaS cluster, refer to [Data location and redundancy](xref:STaaS_features#data-location-and-redundancy).
```

Reviewer: fine for me
